post_installation_configuration/cluster-tasks.adoc

```diff
@@ -201,7 +201,7 @@ For clusters managed by the Assisted Installer, you can add worker nodes by usin
 
 For clusters managed by the multicluster engine for Kubernetes, you can add worker nodes by using the dedicated multicluster engine console.
 
-* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#scale-hosts-infrastructure-env[Scaling hosts to an infrastructure environment]
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.9/html/clusters/cluster_mce_overview#scale-hosts-infrastructure-env[Scaling hosts to an infrastructure environment]
```

Comment:
not sure if this is going to the desired anchor - seems to just go to top of page 
![Screenshot 2023-11-21 at 12 53 46 PM](https://github.com/openshift/openshift-docs/assets/96192119/5e17147d-77cc-4d47-9cb0-7e83e26c8476)

Reply:
That link appears to point to content that no longer exists. I update the link to point to more relevant docs.